test/ui-test/testSuites/suite_status/tst_communityMessageFlows/test.feature

```diff
@@ -0,0 +1,36 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file    test.feature
+# *
+# * \test    Status Desktop - Community Chat Flows
+# * \date    July 2022
+# **
+# *****************************************************************************/
+
+Feature: Status Desktop community
+
+    As a user I want to send messages and interact with channels in a community
+
+
+    Background:
+        Given A first time user lands on the status desktop and generates new key
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        Then the user opens the community portal section
+        Then the user lands on the community portal section
+        When the user creates a community named test_community, with description Community description, intro community intro and outro commmunity outro
+        Then the user lands on the community named test_community
+
+    @mayfail
+    Scenario: User sends a test image
+        When the user sends a test image in the current channel
```

Comment:
Reviewing it again.. I'm missing in all scenarios a `then` validator!!

Reply:
Updated